static/app/components/events/eventEntries.tsx

```diff
@@ -431,7 +433,7 @@ const EventEntries = memo(
             showGroupingConfig={orgFeatures.includes('set-grouping-config')}
           />
         )}
-        {!isShare && hasEventAttachmentsFeature && (
+        {!isShare && !replayId && hasEventAttachmentsFeature && (
```

Comment:
I was looking at what this `hasEventAttachmentsFeature` means and why it's here.

I think we need to do the same check for replays. We have the org-feature already called `session-replay`. So i think it's just a matter of calling `orgFeatures.includes('session-replay')` and guarding.

What this does is, if someone's paying for replays and capturing data that's great, there will be data in the DB. But if they disable the feature (stop paying) then we want to turn the feature off in the UI, they're not charged for the data anymore which will expire as usual.